docs/requirements.txt

```diff
@@ -3,4 +3,56 @@ sphinxcontrib-httpdomain==1.8.0
 sphinx_rtd_theme==0.5.1
 docutils==0.17.1
 sphinx-click==4.3.0
--r ../requirements.txt
+
```

Comment:
Don't know python much, but is this long addition replacing `-r ../requirements.txt` expected?

Reply:
Fixed already.